README.md

```diff
@@ -34,27 +50,28 @@ The task's other options does the following in the order presented below:
 2. Update the default password of the Unity system admin user;
 3. Upload license to Unity.
 
-For updating and querying the Unity system, please refer to the sample task in test-1.yml:
+For updating and querying the Unity system, please refer to the sample task in <ВСТАВЬ>.yml:
 
+    - name: НАПИСАТЬ ПЛЕЙБУК
     - name: Updates and queries
       dellemc_unity:
-        unity_hostname: "192.168.0.202"
-        unity_username: admin
-        unity_password: Password123!
-        unity_updates:
-          - {resource_type: user, name: test1, password: Welcome1!, role: administrator}
-          # - {resource_type: user, id: 'user_test1', role: 'operator'}
-          - {resource_type: remoteSyslog, id: '0', enabled: True, address: '192.168.0.11:515', protocol: 1, facility: 0}
-          - {resource_type: dnsServer, id: '0', addresses: [10.254.66.23, 10.254.66.24]}
-          - {resource_type: ntpServer, id: '0', addresses: [10.254.140.21, 10.254.140.22], rebootPrivilege: 2}
-          # - {resource_type: user, id: 'user_test1', action: 'delete'}
-        unity_password_updates:
-          # - {username: test1, password: Welcome1!, new_password: Welcome2!}
-        unity_queries:
-          # - {resource_type: user, id: 'user_test1', fields: 'role.id'}
-          - {resource_type: remoteSyslog, id: "0", fields: 'address,protocol,facility,enabled'}      # id parameter has to be of the string type
-          - {resource_type: dnsServer, fields: "domain, addresses, origin", page: 1, per_page: 100}
-          - {resource_type: ntpServer, id: "0", fields: addresses}      # id parameter has to be of the string type
+      unity_hostname: "192.168.0.202"
+      unity_username: admin
+      unity_password: Password123!
+      unity_updates:
+        - {resource_type: user, name: test1, password: Welcome1!, role: administrator}
+        # - {resource_type: user, id: 'user_test1', role: 'operator'}
+        - {resource_type: remoteSyslog, id: '0', enabled: True, address: '192.168.0.11:515', protocol: 1, facility: 0}
+        - {resource_type: dnsServer, id: '0', addresses: [10.254.66.23, 10.254.66.24]}
+        - {resource_type: ntpServer, id: '0', addresses: [10.254.140.21, 10.254.140.22], rebootPrivilege: 2}
+        # - {resource_type: user, id: 'user_test1', action: 'delete'}
+      unity_password_updates:
+        # - {username: test1, password: Welcome1!, new_password: Welcome2!}
+      unity_queries:
+        # - {resource_type: user, id: 'user_test1', fields: 'role.id'}
+        - {resource_type: remoteSyslog, id: "0", fields: 'address,protocol,facility,enabled'}      # id parameter has to be of the string type
+        - {resource_type: dnsServer, fields: "domain, addresses, origin", page: 1, per_page: 100}
+        - {resource_type: ntpServer, id: "0", fields: addresses}      # id parameter has to be of the string type
```

Comment:
it's old version, and now it doesn't work

Reply:
Yes, this will be rewrite when all ours version will be merged.